### PR TITLE
January reviews for OpenShift PR1

### PR DESCRIPTION
**Pull request by gellner**
A number of reviews. Only minor changes such as updating documentation links and removing old information.

### CODE REVIEW

File: articles/openshift/oshift-how-expose-nonhttp-services.md [hunk 1]
```diff
@@ -2,10 +2,9 @@
 title: How to expose non-http services externally
 description: Explains how to expose services outside of your OpenShift cluster without using HTTP or HTTPS
 services: openshift
-author: Kieran O'Neill
-reviewer: Gareth Ellner
-lastreviewed: 16/12/2020
-
+author: koeill
```
Comment (benbacon) [inline articles/openshift/oshift-how-expose-nonhttp-services.md:5]: ``
```suggestion
author: koneill
```

File: articles/openshift/oshift-how-restrict-access-to-openshift-routes-by-ip-address.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 title: How to restrict access to OpenShift routes by IP address
 description: Restrict access to OpenShift routes by IP address
 services: openshift
-author: Mudasar Hussain
-reviewer: Steve Mulholland
-lastreviewed: 21/12/2020 10:46:43
+author: mussain
```
Comment (benbacon) [inline articles/openshift/oshift-how-restrict-access-to-openshift-routes-by-ip-address.md:5]:
```suggestion
author: mhussain
```

**Review by benbacon — APPROVED** [review]
Looks good 👍

**Review by suehighmoor — APPROVED** [review]
LGTM

**Pull request merged**